pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -148,6 +164,36 @@ func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
 	}
 }
 
+func loadFileOrDir(fileOrDir string) ([]string, error) {
+	stat, err := os.Stat(fileOrDir)
+	if err != nil {
+		return []string{}, err
+	}
+	if stat.IsDir() {
+		return loadConfigDir(fileOrDir)
+	}
+	return []string{fileOrDir}, nil
+}
+
+func loadConfigDir(dir string) ([]string, error) {
+	if len(dir) == 0 {
+		return []string{}, nil
+	}
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return []string{}, err
+	}
+	result := []string{}
+	for _, file := range files {
+		// Skip hidden files
```

Comment:
Are hidden files on windows used the same way?  I believe "skip hidden files" arose because most Unix editors will drop a temp file into the same directory during editing.

Is there a utility function for checking if a file is hidden?  If so, I agree we should use it, if not I propose not doing so here until such a utility function is created.